src/io/perun/meta.clj

```diff
@@ -1,41 +1,75 @@
 (ns io.perun.meta
   "Utilies for dealing with perun metadata"
-  (:require [boot.core :as boot]))
+  (:require [boot.core :as boot]
+            [clojure.java.io :as io]
+            [clojure.string :as string]
+            [io.perun.core :as perun]))
 
 (def +meta-key+ :io.perun)
 
+(def +global-meta-key+ :io.perun.global)
+
+(def +global-meta-defaults+
+  {:doc-root "public"})
```

Comment:
Okay, I'll give it some thought, then.  My (not really carefully considered) reasoning for putting `:doc-root` in global metadata was that it's a counterpart to `:base-url`, which is also global metadata.  I think this probably holds up, now that I'm thinking about it more. I'd advocate for doing the same thing for `:base-url` and `:doc-root`, whatever that may be.